.github/workflows/release.yaml

```diff
@@ -82,6 +82,11 @@ jobs:
           make operator-push  bundle-push
         env:
           IMG_BASE: ${{ vars.IMG_BASE }}
+      - name: Push Helm Chart to OCI registry
+        shell: bash
+        run: |
+          VERSION=${{ github.event.inputs.tag }}
+          helm push helm-releases/kepler-operator-helm-${VERSION}.tgz oci://quay.io/sustainable_computing_io/charts
```

Comment:
we should remove the helm-releases folder once the chart is pushed and not include the tarball in the release

Reply:
In hindsight we should have used tmp/helm-releases for generating the chart.